Tests/M5Core2TestApp/Program.cs

```diff
@@ -1,94 +1,110 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using nanoFramework.M5Core2;
-using nanoFramework.M5Stack;
-using nanoFramework.Networking;
-using nanoFramework.Runtime.Native;
 using System;
 using System.Diagnostics;
 using System.Threading;
+using nanoFramework.M5Core2;
+using nanoFramework.M5Stack;
 using Console = nanoFramework.M5Stack.Console;
 
+const string StrLB = "LEFT BUTTON PRESSED ";
+const string StrMB = "MIDDLE BUTTON PRESSED";
+const string StrRB = "RIGHT BUTTON PRESSED";
+const string StrTouchEvent = "TOUCHED at (x,y): ";
+const string StrEventID = ", Id=";
+const string StrTouchEventCatagory = "Event Catagory: ";
+const string StrDoubleTouch = "Double Touch.";
+const string StrMove = "Moving...";
+const string StrLiftUp = "Lift Up.";
+const string StrGyro = "Gyroscope (x,y,z): ";
+const string StrAcc = "Accelerometer (x,y,z): ";
+const string StrCpuT = "CPU Temperature: ";
+
+M5Core2.Vibrate = false; // Ensure set to false as can otherwise cause a long vibrate on deploy!
 M5Core2.InitializeScreen();
 
 Debug.WriteLine("Hello from M5Core2!");
+Console.WriteLine("Hello from M5Core2!");
+Vibrate(500);
 
-const string Ssid = "SSID";
```

Comment:
Why do you remove the wifi connection from the test app?

Reply:
I didnt feel it brought anything to the test, and none of the other test apps had it. I can re-add it if you feel it is necessary.

Reply:
It is needed as one of the pin for touch creates some discrepancies when the wifi is on. It's the only way to test that it's still working properly :-)

Reply:
So maybe then add a comment like this and keep the code:

// It is strongly recommended to connect to the wifi while testing the touch interface as wifi is creating discrepancies on the interrupt touch pins.
// This has been managed in the driver but should be tested as well.

Reply:
Ok. Done.